src/Features/CSharp/Portable/Structure/Providers/IndexerDeclarationStructureProvider.cs

```diff
@@ -30,9 +30,13 @@ internal class IndexerDeclarationStructureProvider : AbstractSyntaxNodeStructure
                 return;
             }
 
+            SyntaxNodeOrToken current = indexerDeclaration;
+            var nextSibling = current.GetNextSibling();
+
             spans.AddIfNotNull(CSharpStructureHelpers.CreateBlockSpan(
                 indexerDeclaration,
                 indexerDeclaration.ParameterList.GetLastToken(includeZeroWidth: true),
+                compressEmptyLines: !nextSibling.IsNode || nextSibling.IsKind(SyntaxKind.IndexerDeclaration) || nextSibling.IsKind(SyntaxKind.PropertyDeclaration),
```

Comment:
personally, would be fine with breaking apart indexers/properties.  but if you feel this is better, that's ok with me.

Reply:
I kept them together and added a comment so it's clear it was intentionally implemented (for now at least) with them grouped.